jina/clients/base/websocket.py

```diff
@@ -185,6 +187,7 @@ def _request_handler(
                 result_handler=_result_handler,
                 end_of_iter_handler=_handle_end_of_iter,
                 logger=self.logger,
+                prefetch=prefetch or 0,
```

Comment:
same here

Reply:
I believe it should be `preferch` or `self.args.prefetch` or `self.prefetch` for backwards compatibility?